include/mata/nfa-algorithms.hh

```diff
@@ -26,11 +26,17 @@ namespace Mata {
 namespace Nfa {
 
 /**
- * The following namespace contains methods which would be otherwise in anonymous namespace
- * to make them accessible to users of library. Typically, that are different algorithms for
- * operations such as complement, inclusion, or universality checking.
- * In Nfa interface, there are dispatch functions calling these function according to parameters
- * provided by a user.
+ * NFA algorithms, includes implementations for operations such as complement, inclusion, or universality checking.
```

Comment:
maybe explicitly mention that there are particular implementations for operations such as ....

Reply:
I tried to be brief, but I can expand the comments based on the discussion here.

Reply:
@martinhruska What do you imagine this should look like? Can you give me an example? What is missing here in your opinion?

Reply:
I meant just to declare explicitly that we separate implementation from interface of differerent algortihms using this namespace. Just one sentence would be fine.

Reply:
That could be beneficial. I am OK with adding such sentence.